### PR TITLE
Updates linter for ROS Kinetic and SublimeLinter 4.0.0

### PR DESCRIPTION
**Pull request by Sotilrac**
Updates the linter for ROS Kinetic and SublimeLinter 4.0.0. It should work with Melodic as well, but I haven't tested it.

IDK if you re still maintaining this, but I though you might be interested. Anyhow,. thanks for making this, it's very handy.

### CODE REVIEW

**Comment by lucbettaieb** [comment]
Hoo boy sorry I didn't see this!  Didn't get an email.  I'll set that up.

File: .travis.yml [hunk 1]
```diff
@@ -4,8 +4,6 @@ python:
 # command to install dependencies
 install:
   - pip install flake8
-  - pip install pep257
 # command to run tests
 script:
-  - flake8 . --max-line-length=120
-  - pep257 . --ignore=D202
+  - flake8 . --max-line-length=120
```
Comment (lucbettaieb) [inline .travis.yml]: github likes newlines here

File: README.md [hunk 2]
```diff
@@ -3,9 +3,10 @@ SublimeLinter-roslint
 
 This linter plugin for [SublimeLinter][docs] provides an interface to [roslint](http://wiki.ros.org/roslint). It will be used with files that have the “C++” syntax.
 
-This is for ROS Indigo currently.  You should be able to modify the indigo string inside things to make it work with Jade or other ROS releases. 
+This is tested in ROS Kinetic.
```
Comment (lucbettaieb) [inline README.md:6]: and ROS indigo?

File: linter.py [hunk 3]
```diff
@@ -2,44 +2,23 @@
 # linter.py
 # Linter for SublimeLinter3, a code checking framework for Sublime Text 3
 #
-# Written by Luc Bettaieb, code modefied from NotSqrt
+# By Carlos Asmat. Derived from Luc Bettaieb's work
```
Comment (lucbettaieb) [inline linter.py]: Mind replacing this with 
Authors: Luc Bettaieb, Carlos Asmat
?

File: messages/install.txt [hunk 4]
```diff
@@ -7,4 +7,4 @@ This linter plugin for SublimeLinter provides an interface to roslint.
 Before this plugin will activate, you *must*
 follow the installation instructions here:
 
-https://github.com/lucbettaieb/SublimeLinter-roslint
+https://github.com/Sotilrac/SublimeLinter-roslint
```
Comment (lucbettaieb) [inline messages/install.txt]: This is a fork, no?  Mind leaving it upstream?

**Review by lucbettaieb — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Sotilrac) [reply README.md:6]: I only tested in kinetic. I don't have an Indigo environment, sorry.

File: linter.py (hunk 3 above)
Comment (Sotilrac) [reply linter.py]: Sure, I'll change it.

File: messages/install.txt (hunk 4 above)
Comment (Sotilrac) [reply messages/install.txt]: Yup, sounds good, I just needed it for ppl to install but I rather get it merged.

**Review by lucbettaieb — APPROVED** [review]

**Comment by lucbettaieb** [comment]
Changed the base to be the new master branch, kinetic.

**Pull request merged**